frontend/src/components/GeneInfoSideBar/index.tsx

```diff
@@ -3,14 +3,17 @@ import {
   GeneName,
   GeneSummary,
   GeneSynonyms,
-  GeneSynonymsLabel,
+  Label,
```

Comment:
rename from `GeneSynonymsLabel`, since it's used for more than just synonyms field now